openmls/src/framing/private_message.rs

```diff
@@ -366,15 +366,15 @@ impl PrivateMessage {
         );
 
         let verifiable = VerifiableAuthenticatedContent::new(
-            FramedContentTbs::new(
-                WireFormat::PrivateMessage,
-                self.group_id.clone(),
-                self.epoch,
+            WireFormat::PrivateMessage,
+            FramedContent {
```

Comment:
Is there a reason not to use the `new` function here?

Reply:
I don't think there is a `new` function for `FramedContent`. Would you prefer if I added one?

Reply:
Ah, it's only on the body and the tbs. Nevermind then.